lib/burp/html/issue.rb

```diff
@@ -113,6 +113,9 @@ def method_missing(method, *args)
     def cleanup_request_response_html(source)
       result = source.dup
 
+      # Highlight code
+      result.gsub!(/<span class="HIGHLIGHT">(.+?)<\/span>/, '$${{\1}}$$')
```

Comment:
See this is the issue now we support it in burp but not in Acunetix.

I get that this class is likely proprietary to burp but I still think it's likely time we combine this cleanup into a single place.